### PR TITLE
STYLE: fix pylint: no-else-raise

### PR DESCRIPTION
**Pull request by fabinca**
Issue https://github.com/pandas-dev/pandas/issues/48855. This PR enables pylint type "R" -no-else-raise warning:
- [x] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).

### CODE REVIEW

File: ci/fix_wheels.py [hunk 1]
```diff
@@ -4,7 +4,12 @@
 import zipfile
 
 try:
-    _, wheel_path, dest_dir = sys.argv
+    if len(sys.argv) != 3:
+        raise ValueError(
+            "User must pass the path to the wheel and the destination directory."
+        )
+    wheel_path = sys.argv[1]
+    dest_dir = sys.argv[2]
```
Comment (MarcoGorelli) [inline ci/fix_wheels.py:12]: Thanks @fabinca for your PR

Why does this need changing?

The rest seems fine

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]

File: ci/fix_wheels.py (hunk 1 above)
Comment (fabinca) [reply ci/fix_wheels.py:12]: Without this change it didn't pass the pre-commit checks: 
`ci/fix_wheels.py:11:4: W0632: Possible unbalanced tuple unpacking with sequence: left side has 3 label(s), right side has 2 value(s) (unbalanced-tuple-unpacking)`

Comment (MarcoGorelli) [reply ci/fix_wheels.py:12]: thanks - might be a bug in pylint, I've reported here https://github.com/PyCQA/pylint/issues/7710, let's see what they say

Comment (MarcoGorelli) [reply ci/fix_wheels.py:12]: seems like they'll keep the warning - let's just have an in-line `pylint: disable=` comment then?

**Review by MarcoGorelli — APPROVED** [review]
actually, this is fine, thanks @fabinca

**Comment by jbrockmendel** [comment]
Just saw this failing the CI in a PR.  I find this more annoying than helpful.

**Comment by MarcoGorelli** [comment]
> Just saw this failing the CI in a PR. I find this more annoying than helpful.

Fixed in https://github.com/pandas-dev/pandas/pull/49537 , apologies

I think it's pretty neat, but we can turn this check back off if you prefer

**Pull request merged**